sketch_oct9a.ino

```diff
@@ -201,7 +180,98 @@ void loop() {
   if (alt.getStatus() == Altimeter::ACTIVE) {
     last_alt = alt.getAltitude();
   }
+}
+
+void sendDataToRadio() {
+  Frame f = createFrame(
+    last_coords,
+    last_accel,
+    last_gyro,
+    last_alt
+#if CAPSULE == 2
+    ,
+    last_voc,
+    last_humid,
+    last_temp
+#endif
+  );
+
+  // In theory, this should send the data to the radio.
+  // In practice, this is completely untested. If nothing is connected to the UART pins, however,
+  // this doesn't do anything, so it's harmless to leave in for now.
+  Serial1.write(
+    (char*)&f,
+    FRAME_SIZE()
+  );
+}
+
+void setup() {
+  Serial1.begin(230400);
```

Comment:
Put a comment in the code on this line that this is the actual rate we will be sending at. You could even do this in a 'DEFINE' statement closer to the top of the file.